deps/wazuh_testing/wazuh_testing/qa_docs/lib/config.py

```diff
@@ -32,153 +36,194 @@ def __init__(self, *args):
         self.test_fields = _fields()
         self.test_cases_field = None
 
-        try:
-            with open(args[0]) as fd:
-                self._config_data = yaml.safe_load(fd)
-        except:
-            logging.error("Cannot load config file")
-            raise Exception("Cannot load config file")
-
-        self._read_function_regex()
-        self._read_output_fields()
-        self._read_test_cases_field()
-        self._read_documentation_path()
-        self._read_include_paths()
-        self._read_include_regex()
-        self._read_group_files()
-        self._read_ignore_paths()
+        self.__load_config_file(args[0])
+        self.__read_function_regex()
+        self.__read_output_fields()
+        self.__read_test_cases_field()
+        self.__read_include_paths()
+        self.__read_include_regex()
+        self.__read_group_files()
+        self.__read_ignore_paths()
 
         if len(args) >= 3:
-            self.documentation_path = args[2]
+            self.__set_documentation_path(args[2])
         if len(args) == 4:
             # It is called with a single test to parse
             self.mode = mode.SINGLE_TEST
             self.test_name = args[3]
-            self._read_test_info()
-            self._read_module_info()
+            self.__read_test_info()
+            self.__read_module_info()
+
+
+    def __load_config_file(self, file):
+        try:
+            Config.LOGGER.debug('Loading config file')
+            with open(file) as config_file:
+                self._config_data = yaml.safe_load(config_file)
+        except:
+            raise QAValueError('Cannot load config file', Config.LOGGER.error)
+
+    def __read_test_info(self):
+        """Reads from the config file the keys to be printed from module info.
 
+        This functionality is used to print any custom field(s) you want.
+        You can use it if you only need a few fields to parse when a single test is run.
+        
+            For example you have this in your config.yaml:
 
-    def _read_test_info(self):
-        '''
-        brief: Reads from the config file the keys to be printed from test info
-        '''
+            Test info:
+                - test_wazuh_min_version: wazuh_min_version
+        """
+        Config.LOGGER.debug('Reading test info from config file')
```

Comment:
```suggestion
        Config.LOGGER.debug('Reading test info from the config file')
```

Reply:
Done in: https://github.com/wazuh/wazuh-qa/commit/86e5045beb49f8e91dbb42b6e59f8d3164a57523